doc/network.tex

```diff
@@ -244,9 +245,9 @@ \chapter{Overview}
 \section{High level requirements}
```

Comment:
This is a style I think people will relate to "the user stories" that motivate things!